lib/Doctrine/Parser/Xml.php

```diff
@@ -46,7 +46,7 @@ public function dumpData($array, $path = null, $charset = null)
     {
         $data = self::arrayToXml($array, 'data', null, $charset);
 
-        return $this->doDump($data, $path);
+        return $this->doDump((string) $data, $path);
```

Comment:
`arrayToXml` can return `false` so just forcing that to a string here (which would preserve current behavior).

Probably should throw an exception or something if the data can't be converted to XML.